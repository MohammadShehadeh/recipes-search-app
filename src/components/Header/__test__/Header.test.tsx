import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { Header } from '..';

describe('Header', () => {
  it('should render to match snapshot', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render Header correctly', () => {
    render(<Header />);
    const headerElement = screen.getByText(/Search Recipes/i);
    expect(headerElement).toBeInTheDocument();
  });
});
