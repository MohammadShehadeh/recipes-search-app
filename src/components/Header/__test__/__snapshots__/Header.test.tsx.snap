// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render to match snapshot 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="wrapper main"
    >
      <a
        className="logo"
        href="/"
      >
        Search Recipes
      </a>
    </div>
  </div>
</header>
`;
